Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -571,4 +571,12 @@ extension ChartDataSet: RangeReplaceableCollection {
         entries.removeAll(keepingCapacity: keepCapacity)
         notifyDataSetChanged()
     }
+    
+    public func replaceSubrange<C>(
+        _ subrange: Range<Index>,
```

Comment:
I think if we just use `Swift.Range<Index>` here we don't have to rename the other Range class. I think it would be a better fix for the problem.